go.mod

```diff
@@ -137,7 +137,7 @@ require (
 	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
 	github.com/prometheus/common v0.7.0
 	github.com/prometheus/procfs v0.0.11
-	github.com/prometheus/prometheus v2.5.0+incompatible
```

Comment:
Thanks for removing the unneeded changes here!

This looks like if we are downgrading from 2.5.0 to 1.8.2, is this expected?

Reply:
Prometheus doesn't adopt go modules properly, that's why I had to use Release branch, to be able to use OpenMetrics features.

Reply:
@jsoriano this is the same reason why we use `v7` on the import paths. prometheus doesnt do that. exemplar support was added only a couple of releases ago.

Reply:
Oh, I see, LGTM then.

For the record:
* github.com/prometheus/prometheus not defined as /v2 in their [go.mod](https://github.com/prometheus/prometheus/blob/v2.29.2/go.mod#L1).
* https://github.com/prometheus/prometheus/issues/8852
* https://github.com/golang/go/issues/35732